flexmeasures/data/models/reporting/aggregatator.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import annotations
+
+from typing import Any
+from datetime import datetime, timedelta
+
+import timely_beliefs as tb
+import pandas as pd
+
+from flexmeasures.data.models.reporting import Reporter
+from flexmeasures.data.schemas.reporting.aggregation import (
+    AggregatorSchema,
+    AggregationMethod,
+)
+from flexmeasures.data.models.time_series import TimedBelief
+from flexmeasures.utils.time_utils import server_now
+
+
+class AggregatorReporter(Reporter):
+    """This reporter applies an aggregation function to multiple sensors"""
+
+    __version__ = "1"
+    __author__ = None
+    schema = AggregatorSchema()
+    transformations: list[dict[str, Any]] = None
```

Comment:
Are these now used? I would have expected the `AggregatorReporter` to subclass from `PandasReporter` and set its `transformations`. Looks like you chose a different way of executing the transformations, in which case this property could probably go?

Reply:
Sure! I chose to subclass `Reporter` because I wanted to use `pandas.concat` function, basically. Also, I think it would be equal (or more) size and much less readable.